Docs/Doxyfile

```diff
@@ -790,7 +790,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ../Source/
```

Comment:
I was confused why we add the Tools here for doxygen source parsing. Are there C++ files in `Tools/` that we want to pick up?

Reply:
clarified offline: Doxygen can actually parse the python scripts in `Tools/LibEnsemble` so we can add the `run_warpx` function doc string here.